__mocks__/generators/datasets.ts

```diff
@@ -0,0 +1,86 @@
+/*
+* This program and the accompanying materials are made available under the terms of the *
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at *
+* https://www.eclipse.org/legal/epl-v20.html                                      *
+*                                                                                 *
+* SPDX-License-Identifier: EPL-2.0                                                *
+*                                                                                 *
+* Copyright Contributors to the Zowe Project.                                     *
+*                                                                                 *
+*/
+
+import { ZoweDatasetNode } from "../../src/dataset/ZoweDatasetNode";
+import * as vscode from "vscode";
+import * as imperative from "@zowe/imperative";
+import * as globals from "../../src/globals";
+
+export function generateDatasetSessionNode(session: imperative.Session, profile: imperative.IProfileLoaded) {
+    const datasetNode = new ZoweDatasetNode("sestest", vscode.TreeItemCollapsibleState.Expanded,
+        null, session, undefined, undefined, profile);
+    datasetNode.contextValue = globals.DS_SESSION_CONTEXT;
+
+    return datasetNode;
+}
+
+export function generateDatasetTree(sessionNode: ZoweDatasetNode, treeView: any): any {
```

Comment:
Typing?

Reply:
Tree View doesn't fully match the original type, so if it's okay I would keep it as any.